src/vis/vis.js

```diff
@@ -264,7 +264,7 @@ var VisModel = Backbone.Model.extend({
   },
 
   _getAnalysisNodeModels: function () {
-    return this._analysisCollection.models;
+    return this._getUniqueAnalysesNodes();
```

Comment:
There's no reason to have both `_getAnalysisNodeModels` and `_getUniqueAnalysesNodes`. I'd just keep `_getUniqueAnalysesNodes`.